app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
+
```

Comment:
Code looks good 👍  Zech, I didn't know about `Serializer`. ActiveModel's got it all it seems.